### PR TITLE
fixed small typo

### PR DESCRIPTION
**Pull request by tmeralus**
changed uprate to upgrade on line 12

### CODE REVIEW

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks for this PR @tmeralus 

I think perhaps we need a different fix where we replace “uprate” with an explanatory phrase.
“uprate” is used here in its dictionary meaning, but I'm guessing that we could pick something that is less easily confused with “upgrade”.

**Comment by tmeralus** [comment]
> Thanks for this PR @tmeralus
> 
> I think perhaps we need a different fix where we replace “uprate” with an explanatory phrase.
> “uprate” is used here in its dictionary meaning, but I'm guessing that we could pick something that is less easily confused with “upgrade”.

Forgive me for not checking that definition and allow me to  suggest the word "advance" or "enhance"

File: content/en/docs/setup/production-environment/_index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ no_list: true
 A production-quality Kubernetes cluster requires planning and preparation.
 If your Kubernetes cluster is to run critical workloads, it must be configured to be resilient.
 This page explains steps you can take to set up a production-ready cluster,
-or to uprate an existing cluster for production use.
+or to upgrade an existing cluster for production use.
```
Comment (bradtopol) [inline content/en/docs/setup/production-environment/_index.md:12]: good catch!

Comment (bradtopol) [inline content/en/docs/setup/production-environment/_index.md:12]: I agree uprate is confusing here.

Comment (bradtopol) [reply content/en/docs/setup/production-environment/_index.md:12]: @tmeralus @sftim How about using the word "promote" ?
/assign

**Comment by bradtopol** [comment]
I unassigned myself since I am going on vacation.  Assign @sftim since he has been engaged on this.
/assign @sftim

File: content/en/docs/setup/production-environment/_index.md (hunk 1 above)
Comment (reylejano) [inline content/en/docs/setup/production-environment/_index.md:12]:
```suggestion
or to promote an existing cluster for production use.
```
I like Brad's suggestion. What do you think @tmeralus ?

Comment (tmeralus) [reply content/en/docs/setup/production-environment/_index.md:12]: That is more clear 👌🏾 I like that

Comment (reylejano) [reply content/en/docs/setup/production-environment/_index.md:12]: @tmeralus Feel free to commit the change on your end

**Review by tmeralus — COMMENTED** [review]
updated wording in kubernetes docs

**Comment by tmeralus** [comment]
updated documentation.

**Pull request merged**